src/main/scala/org/hammerlab/guacamole/distributed/PileupFlatMapUtils.scala

```diff
@@ -6,15 +6,16 @@ import org.hammerlab.guacamole.loci.partitioning.LociPartitioning
 import org.hammerlab.guacamole.pileup.Pileup
 import org.hammerlab.guacamole.reads.MappedRead
 import org.hammerlab.guacamole.readsets.PerSample
-import org.hammerlab.guacamole.reference.{ReferenceGenome, _}
+import org.hammerlab.guacamole.reference.{ContigSequence, ReferenceGenome}
 import org.hammerlab.guacamole.windowing.SlidingWindow
 
 import scala.reflect.ClassTag
 
 object PileupFlatMapUtils {
   /**
    * Helper function. Given optionally an existing Pileup, and a sliding read window return a new Pileup at the given
-   * locus. If an existing Pileup is given as input, then the result will share elements with that Pileup for efficiency.
```

Comment:
I thought we were agreed that >120 chars was fair game for breaking onto multiple lines.